main/telemetry.js

```diff
@@ -49,7 +50,12 @@ setInterval(() => {
   point.stringField('version', pkg.version)
   point.tag('station', 'desktop')
   point.tag('platform', platform())
-  point.tag('arch', arch())
+  // `os.arch()` returns `x64` when running in a translated environment on ARM64
+  // machine, e.g. via macOS Rosetta. We don't provide Apple `arm64` builds yet.
+  // As a result, Stations running on Apple Silicon were reporting `x64` arch.
+  // To fix the problem, we detect `arm64` translation and report a different
+  // architecture.
+  point.tag('arch', app.runningUnderARM64Translation ? 'arm64' : arch())
```

Comment:
Discussion point: do we want to report both the build-arch (x64) and machine-arch (arm64) to measure how many people are running a version built for a different architecture?

It does not matter right now (we provided only x64 builds for macOS, all Macs are running the emulation), but it may become interesting after we implement support for Apple Silicon - #35.

Reply:
This is a cool idea for the future, hopefully we'll add native builds before we need to implement it I think